src/Concurrency/Info.hpp

```diff
@@ -51,10 +58,13 @@ unsigned int getWorkerId();
 template<>
 inline unsigned int getWorkerId<Executor::OPENMP>()
 {
+#ifdef ENABLE_OPENMP
   return omp_get_thread_num();
+#else
+  return 1;
```

Comment:
I think this should return zero, since in OMP the threads are numbered from 0 to omp_get_num_threads()-1.

Reply:
Agreed.

Reply:
Did I mix it with `omp_get_num_threads()`? I'll fix.